CODEOWNERS

```diff
@@ -5,6 +5,9 @@
 
 /node/ @sea212
 /runtime/ @sea212
```

Comment:
Doesn't this conflict with `/runtime/src/weights/*`?

Reply:
The last matching rule in the CODEROWNERS file is always applied (see reference in the PR description).
In this case, it will:

- Accept runtime/common/src/mod.rs
- Decline everything else in runtime/common/src/mod.rs
- Accept everything remaining in runtime

Reply:
I see now.